<launch>

    <node pkg="robot_tensorflow" name="ObjectDetectionDemo" type="ObjectDetectionDemo.py" output="screen">
        <param name="image_topic" value="/usb_cam/image_raw" />
        <param name="model_path" value="$(find robot_tensorflow)/object_detection" />
    </node> 
</launch>
